packages/web-components/src/toolbar/toolbar.spec.md

```diff
@@ -0,0 +1,74 @@
+# Toolbar
+
+## Component Description
+
+A toolbar is a container for grouping a set of controls, such as buttons, menubuttons, or checkboxes.
+
+When a set of controls is visually presented as a group, the toolbar role can be used to communicate the presence and purpose of the grouping to screen reader users. Grouping controls into toolbars can also be an effective way of reducing the number of tab stops in the keyboard interface.
+
+## Design Spec
+
+[Link to Design Spec in Figma](https://www.figma.com/file/QfHyx6ApwFEcw5q4pM9CSZ/Toolbar?node-id=2351-6672&t=MuajdFxoCBjwlK4w-0)
+
+[Link to Fluent 2 Storybook](https://master--628d031b55e942004ac95df1.chromatic.com/?path=/docs/components-toolbar--default)
+
+## Engineering Spec
+
+### Inputs
+
+- @attr appearance: "static" | "contextual"
+- @attr size: "small" "medium" "large" | "medium"
+- @attr vertical: boolean | false
+  -- FAST uses `orientation`: enum { horizontal, vertical }
```

Comment:
@chrisdholt Does it help to use my approach on line 21 versus leveraging existing attribute from FAST on line 22? If I put my Chris Holt™ baseball cap on, I would think that an implementation using a Boolean would be more concise. However, as I am extending a FAST component, it may be that introducing a Boolean means adding complexity.

Reply:
Based on our working model we should align to the FAST Foundation implementation here. Adding vertical would keep both API surfaces which would be confusing because they accomplish the same thing.